commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/integration/SerializationService.kt

```diff
@@ -15,13 +16,15 @@ interface SerializationService {
 
     fun serializeFileList(files: Sequence<BioFile>, targetFormat: SubFormat, outputStream: OutputStream)
```

Comment:
Do we still need this method?

Reply:
we actually need this for example when we serialize a table (limited number of files)